packages/firestore/test/integration/api/query.test.ts

```diff
@@ -1299,6 +1299,75 @@ apiDescribe('Queries', (persistence: boolean) => {
     return;
   }
 
+  // TODO(orquery): Enable this test when prod supports OR queries.
```

Comment:
Does it pass against the emulator?

Reply:
Everything was passing against the emulator when this was originally published.